test/parallel/test-dgram-setTTL.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+const socket = dgram.createSocket('udp4');
+
+socket.bind(common.PORT);
+socket.on('listening', function() {
```

Comment:
Since the entire test is in the callback, we can wrap it with `common.mustCall`.